lib/ansible/modules/cloud/vmware/vmware_cluster_drs.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2015, Joseph Callen <jcallen () csc.com>
+# Copyright: (c) 2018, Ansible Project
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_cluster_drs
+short_description: Manage DRS on VMware vSphere clusters
+description:
+    - This module can be used to manage DRS on VMware vSphere clusters.
+    - All values and VMware object names are case sensitive.
+version_added: '2.9'
+author:
+- Joseph Callen (@jcpowermac)
+- Abhijeet Kasurde (@Akasurde)
+requirements:
+    - Tested on ESXi 5.5 and 6.5.
+    - PyVmomi installed.
+options:
+    cluster_name:
+      description:
+      - The name of the cluster to be managed.
+      type: str
+      required: yes
+    datacenter:
+      description:
+      - The name of the datacenter.
+      type: str
+      required: yes
+      aliases: [ datacenter_name ]
+    enable_drs:
+      description:
+      - If set to C(yes), will enable DRS when the cluster is created.
+      type: bool
+      default: 'no'
+    drs_enable_vm_behavior_overrides:
+      description:
+      - Determines whether DRS Behavior overrides for individual virtual machines are enabled.
+      - If set to C(True), overrides C(drs_default_vm_behavior).
+      type: bool
+      default: True
+      version_added: 2.8
+    drs_default_vm_behavior:
+      description:
+      - Specifies the cluster-wide default DRS behavior for virtual machines.
+      - If set to C(partiallyAutomated), then vCenter generate recommendations for virtual machine migration and
```

Comment:
I'm not sure I have this right, but here's my guess at what this means:
```suggestion
      - If set to C(partiallyAutomated), vCenter generates recommendations for virtual machine migration and
```